src/uint256.cpp

```diff
@@ -20,7 +20,7 @@ base_blob<BITS>::base_blob(const std::vector<unsigned char>& vch)
 template<unsigned int BITS>
 base_blob<BITS>::base_blob(const std::array<unsigned char, WIDTH>& vch)
 {
-    memcpy(data, &vch[0], sizeof(data));
+    memcpy(data, &vch, sizeof(data));
```

Comment:
`std::array` does not overload operator `&`. That mean this one will return the address of `std::array` instead of the underlying array that stored the data. If the implementation put the underlying array as the first member and compiler does not put any implementation specific data before it then it will work but it is still dangerous. So better to use `vch.data()` instead.

Reply:
The actual issue was a missing header (error message didn't indicate as such). reverted to original and added this header.